### ISSUE TITLE
Clarify whether a project can deal with more than 1 dataset

### ISSUE THREAD

**Issue by philippotto** [scalableminds/webknossos#5809]
## Detailed Description
When creating a project, one doesn't need to specify a dataset which suggests that a project can be used for multiple datasets. However the project list supports a "View" button which opens a `CompoundAnnotation`. I assume an error will be thrown if this edge case is provoked?

The UI could explain limitations/recommendations.

## Context

* [ ] Specific to long-running jobs (set `jobsEnabled=true` in `application.conf`)
* [ ] Specific to webKnossos.org (set `isDemoInstance=true` in `application.conf`)

**Comment by fm3** [comment]
Yes, I think you can create tasks for different datasets in the same project. The CompoundAnnotation merging will fail.
Not sure what the best model / restrictions should be.

**Comment by fm3** [comment]
Related: #5447
